docs-core/src/main/java/com/sismics/docs/core/model/jpa/Document.java

```diff
@@ -47,6 +47,12 @@ public class Document implements Loggable {
     @Column(name = "DOC_TITLE_C", nullable = false, length = 100)
     private String title;
     
+    /**
+     * GPA range.
+     */
+    @Column(name = "DOC_GPA_C", nullable = false, length = 20)
+    private String gpa = "3.7-4.0";
```

Comment:
not sure why set to this

Reply:
This had been fixed through (uncommited) merged conflicts. The dev branch is now updated so gpa's default value is "N/A".